packages/terra-menu/tests/nightwatch/menu/menu-spec.js

```diff
@@ -61,4 +62,14 @@ module.exports = resizeTo(['tiny', 'small', 'medium', 'large', 'huge', 'enormous
       .assert.hidden('div[class*="_header"] > div:last-child > button[class*="_header-button"]')
       .keys([browser.Keys.ESCAPE]);
```

Comment:
Add tests for up and down navigation?

Reply:
I haven't been able to find a way to verify that the focus changes on up/down... thoughts?

Reply:
Ah. I have yet to find a good way to determine what element on the page has focus either.

Reply:
Here is a [custom nightwatch assertion for testing focus](https://gist.github.com/Rykus0/fa62d641dfeb75316d04). We don't need to add in it to this review, but it would be nice to add this in the future as we get into more focus management on various components.

Reply:
That's pretty neat! I do think there is value to adding it. There are a few components were testing focus would be important (this one, time input, slide group). I think adding it to the toolkit would be a nice to have.

Reply:
I halfway implemented the hasFocus assertion in a local branch, need to get it out for review.